firebase-app-distribution/src/test/java/com/google/firebase/appdistribution/AabUpdaterTest.java

```diff
@@ -72,6 +77,15 @@ public void setup() throws IOException, FirebaseAppDistributionException {
 
     FirebaseApp.clearInstancesForTest();
 
+    FirebaseApp firebaseApp =
+        FirebaseApp.initializeApp(
+            ApplicationProvider.getApplicationContext(),
+            new FirebaseOptions.Builder()
+                .setApplicationId(TEST_APP_ID_1)
+                .setProjectId(TEST_PROJECT_ID)
+                .setApiKey(TEST_API_KEY)
+                .build());
```

Comment:
In case you missed my other comment, you don't need to a create a `FirebaseApp` here. You can just inject `ApplicationProvider.getApplicationContext()` into the constructor, below.

Reply:
I did miss this comment. Will fix now.